packages/core/src/common/keybinding.ts

```diff
@@ -170,7 +171,20 @@ export class KeybindingRegistry {
      * @param commandId The ID of the command for which we are looking for keybindings.
      */
     getKeybindingsForCommand(commandId: string): Keybinding[] {
-        return this.commands[commandId] || [];
+        const result: Keybinding[] = [];
+
+        for (let scope = KeybindingScopes.WORKSPACE; scope >= KeybindingScopes.DEFAULT; scope--) {
+            this.keymaps[scope].forEach(binding => {
+                if (binding.commandId === commandId) {
+                    result.push(binding);
+                }
+            });
+
+            if (result.length > 0) {
+                return result;
+            }
+        }
+        return result;
```

Comment:
@hexa00 Not even used at this moment, but should I go further and return all keybindings (shadowed or not) associated with a command id? I know this might be useful in later iterations for the editor and whatnot, but maybe we could add that further down the line?